src/login.coffee

```diff
@@ -35,6 +35,18 @@ define ['jquery', 'primedia_events', 'jquery.cookie'], ($, events) ->
 
         $("a.logout").click (e) => @_logOut e
 
+    _prefillAccountName: ($div) ->
```

Comment:
What do you think about passing in `'input[name="new_first_name"]'` and `'input[name="new_last_name"]'` as arguments? Then it would separate the lib from the app.